src/configure.ac

```diff
@@ -61,6 +61,17 @@ else
 fi
 AC_SUBST(AUTODIRS)
 
+# Switch sysconfig 'platlib' path from
+# /usr/local/lib/python3.11/dist-packages to
+# /usr/lib/python3/dist-packages, see
+# <URL: https://bugs.debian.org/1080668 >.  This affect SITEPY and
+# might also affect AX_PYTHON and AX_PYTHON_DEVEL.
+case "$(lsb_release -is)" in
+    Debian|Ubuntu)
+	export DEB_PYTHON_INSTALL_LAYOUT=deb
```

Comment:
Speaking with a Debian hat, without knowing anything about linuxcnc's build system:

This is appropriate for a `/usr` install, but not a `/usr/local` install.

I suggested that in the context of a Debian packaging patch, not something upstreamable.

Reply:
I changed the approach and moved the environment variable to debian/rules(.in) instead.